pack3d/pack3d_test.go

```diff
@@ -0,0 +1,40 @@
+package pack3d_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Authentise/pack3d/pack3d"
+)
+
+func TestCoPack(t *testing.T) {
+	config, err := pack3d.ParseConfig("../tests/fixtures/copack.json")
+
+	fmt.Printf("%v", config)
+	if err != nil {
+		t.Fatalf("Failed to load config: %s", err)
+	}
+
+	_, err = pack3d.Pack(config)
```

Comment:
anyway we could assert/validate the packing output here, if it matters?

Reply:
Added this in the other PR!